tests/test_mibig_downloader.py

```diff
@@ -0,0 +1,52 @@
+import shutil
+import tempfile
+from pathlib import Path
+import pytest
+from nplinker.genomics import mibig
+
+
+class TestDownloadAndExtractMibigMetadatas():
+
+    @pytest.fixture
+    def temppath1(self):
+        temppath = tempfile.mkdtemp()
+        yield temppath
+        shutil.rmtree(temppath)
+
+    @pytest.fixture
+    def temppath2(self):
+        temppath = tempfile.mkdtemp()
+        yield temppath
+        shutil.rmtree(temppath)
```

Comment:
Actually pytest has a built-in fixture for temporary paths: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Nice! will take a look to see if it can replace some of my code